src/pages/integrations/platforms/vercel.mdx

```diff
@@ -0,0 +1,380 @@
+import { Tag } from '@/components/Tag'
+
+export const description = 'Integrate Phase with Vercel'
+
+<Tag variant="small">INTEGRATE</Tag>
+
+# Vercel
+
+You can use Phase to sync secrets with your Vercel projects.
+
+<Warning>
+  When secret syncing is enabled, secrets stored inside Phase will be treated as
+  a source of truth. Any secrets on target service will be overwritten /
+  deleted. Please import your secrets in Phase before continuing.
+</Warning>
+
+### Prerequisites
+
+- Sign up for the [Phase Console](/quickstart) and [create an App](/console/apps#create-an-app).
+- Enable Server-side Encryption (SSE) for the App from the [Settings](/console/apps#settings) tab.
+
+## Step 1: Authentication
+
+### Create a Vercel Token
+
+1. Go to Vercel's [Dashboard](https://vercel.com/login) and log in.
+
+2. Click on your user icon in the top right and select **Account Settings**.
+
+![Vercel dashboard](/assets/images/platform-integrations/vercel/1.png)
+
+3. In account settings, click on **Tokens** from the menu bar in the left.
+
+![Vercel dashboard account settings](/assets/images/platform-integrations/vercel/2.png)
+
+4. Create a new **Token** with a descriptive name (eg. `phase-console-vercel-sync`) , set the **Scope** to `Full Account` and **Expiration** to `No Expiration` and click **Create**. 
```

Comment:
```suggestion
4. Create a new **Token** with a descriptive name (eg. `phase-console-vercel-sync`) , set the **Scope** to `Full Account`, choose an **Expiration** and click **Create**. (Note: If you choose an expiry option other than `No Expiration`, you will need to manually update this token in Phase when it expires.)
```

Don't think we should ask users select a permanent token by default, let them decide how to handle it.